etc/non-ecs-schema.json

```diff
@@ -37,7 +37,9 @@
     "o365.audit.NewValue": "keyword",
     "o365audit.Parameters.ForwardTo": "keyword",
     "o365audit.Parameters.ForwardAsAttachmentTo": "keyword",
-    "o365audit.Parameters.RedirectTo": "keyword"
+    "o365audit.Parameters.RedirectTo": "keyword",
```

Comment:
do you have an example event? If you add the dataset or module, we should not need to add to this schema

Reply:
Not sure what you mean; If I remove the mappings the test complains:
```
E               kql.errors.KqlParseError: Error at line:2,column:3
E               Unknown field
E               event.provider:rds.amazonaws.com and event.action:ModifyDBSnapshotAttribute and
E                 aws.cloudtrail.flattened.request_parameters.attributeName:"restore" and
E                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E               
E               Try adding event.module or event.dataset to specify beats module
E               
E               stack: 7.13.0, beats: 7.13.2, ecs: 1.9.0
```

Reply:
I tried with:
*  `event.module:aws and event.dataset:aws.cloudtrail`
* `event.module:aws`
* `event.dataset:aws.cloudtrail`

All three fail with the same error. Hence adding the fields

Reply:
@SHolzhauer in your stack (Kibana, timelines, etc.) can you grab the json of the event(s) that this query will identify and paste it here?

Reply:
I just saw this right now, and from what I understood, it seems the case of adding it to the non-ecs-schema.

[Here](https://github.com/elastic/integrations/blob/e85849eee06bec5ced1c1c4a2fe2325ec4ead7d3/packages/aws/data_stream/cloudtrail/fields/fields.yml#L151) is the field mapping for `request_parameters`, which defines it as flattened, which reading the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/flattened.html), looks like it dynamically parses the JSON object (its value) and turns the keys of the object into subfields.